tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -124,7 +124,7 @@
     "## Single-Qubit Gates <a name=\"single_gates\"/>\n",
```

Comment:
<p>I am not sure there is a reason to mention this special case any more since u2 gate is deprecated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Should be addressed now - removed the original u2 portion instead of converting it to a second u example.